src/Ouroboros-Mini_Protocols-Chain_Sync.thy

```diff
@@ -0,0 +1,226 @@
+section \<open>Chain-Sync Mini-Protocol\<close>
```

Comment:
I would say “Chain Synchronization” instead of “Chain-Sync” here. I know that *The Shelley Networking Protocol* seems to use “chain-sync” as the name of the protocol. However, I don’t like that. “Chain-sync” and “block-fetch”, with their abbreviations and their hyphens, look rather like identifiers to me, but then they are used like normal words. I suggest that we use proper, long names like “chain synchronization” in prose and the short versions in identifiers.

Reply:
Fixed in fc43b919b7c9c70ff015fe127d2ee031f5ec2109, please check.

Reply:
Looks pretty nice to me. :slightly_smiling_face: